demo/standalone/util.rb

```diff
@@ -99,17 +99,19 @@ def call(env)
 end
 
 # Add persistent storage volumes
-def attach_volumes(node, num_volumes, volume_size)
+def attach_volumes(node, disk_sizes)
 
     node.vm.provider :virtualbox do |v, override|
```

Comment:
not with this PR at least.  only virtualbox is so far supported, but you could open an issue to add vmware support too.

note this standalone vagrant is used less often than kubernetes, which there is vmware support in the coreos-kubernetes repo.